plugins/module_utils/network/eos/argspec/ospfv2/ospfv2.py

```diff
@@ -0,0 +1,326 @@
+                        },
+                        "max_lsa": {
+                            "options": {
+                                "count": {"type": "int"},
+                                "ignore_count": {"type": "int"},
+                                "ignore_time": {"type": "int"},
+                                "reset_time": {"type": "int"},
+                                "threshold": {"type": "int"},
+                                "warning": {"type": "bool"},
+                            },
+                            "type": "dict",
+                        },
+                        "max_metric": {
+                            "options": {
+                                "router_lsa": {
+                                    "options": {
+                                        "set": {"type": "bool"},
+                                        "include_stub": {"type": "bool"},
+                                        "on_startup": {
+                                            "options": {
+                                                "wait_period": {"type": "int"}
+                                            },
+                                            "type": "dict",
+                                        },
+                                        "summary_lsa": {
+                                            "options": {
+                                                "max_metric_value": {
+                                                    "type": "int"
+                                                },
+                                                "set": {"type": "bool"},
+                                            },
+                                            "type": "dict",
+                                        },
+                                        "external_lsa": {
+                                            "options": {
+                                                "max_metric_value": {
+                                                    "type": "int"
+                                                },
+                                                "set": {"type": "bool"},
+                                            },
+                                            "type": "dict",
+                                        },
+                                    },
+                                    "type": "dict",
+                                }
+                            },
+                            "type": "dict",
+                        },
+                        "maximum_paths": {"type": "int"},
+                        "mpls_ldp": {"type": "bool"},
+                        "networks": {
+                            "elements": "dict",
+                            "options": {
+                                "area": {"type": "str"},
+                                "mask": {"type": "str"},
+                                "network_address": {"type": "str"},
```

Comment:
If we have decided to restrict the granularity for the deleted operation, then we might want to look out 
for mandatory params (required: True).